client/src/main/java/se/fortnox/reactivewizard/client/HttpClient.java

```diff
@@ -590,6 +590,24 @@ protected String getPath(Method method, Object[] arguments, JaxRsMeta meta) {
         return result;
     }
 
+    /**
+     * Recursive function getting all declared fields from the passed in class and its ancestors
+     *
+     * @param clazz the clazz
+     * @param fields the fields found so far.
+     *
+     * @return list of fields
+     */
+    private List<Field> getDeclaredFieldsFromClassAndAncestors(Class clazz, List<Field> fields) {
+        fields.addAll(asList(clazz.getDeclaredFields()));
```

Comment:
If we have apache collections, we could do a union of two sets without mutating the provided list :) return SetUtils. union(a, b) ;

Reply:
👍